README.md

```diff
@@ -178,11 +190,8 @@ Support us with a monthly donation and help us continue our activities. [[Become
 
 MIT
 
-
-
 [![Preact](https://i.imgur.com/YqCHvEW.gif)](https://preactjs.com)
 
-
 [preact/compat]: https://github.com/preactjs/preact/tree/master/compat
 [hyperscript]: https://github.com/dominictarr/hyperscript
-[DevTools]: https://github.com/preactjs/preact-devtools
+[devtools]: https://github.com/preactjs/preact-devtools
```

Comment:
I'd prefer keeping this as it was.

There's no functional difference but "DevTools" matches the capitalization of the point of usage. I find that helps in identifying out-dated links, as can occur over time.